README.md

```diff
@@ -129,6 +129,10 @@ brew bundle
   git commit -am "Update README.md"
   ```
 
+7. Close app from unauthorized viewers
```

Comment:
Not sure this is a right place for this note. Would you like to create separate section to describe `ENV` variables we use in the application?